src/MOLFiniteDifference/MOL_discretization.jl

```diff
@@ -31,7 +33,7 @@ function SciMLBase.symbolic_discretize(pdesys::ModelingToolkit.PDESystem,discret
 
     # Build symbolic variables
     indices = CartesianIndices(((axes(s)[1] for s in space)...,))
-    depvars = map(pdesys.depvars) do u
+    depvarsdisc = map(depvars) do u
```

Comment:
what does the name mean?

Reply:
discretized - I found it confusing that depvars != pdesys.depvars

Reply:
good point, yes it was a consequence of writing too fast. Thanks for correcting it.